packages/store/tests/select.spec.ts

```diff
@@ -183,6 +263,47 @@ describe('Select', () => {
     });
   }));
 
+  it('should be throw when use reserved select property in class', async(() => {
```

Comment:
`should throw an exception if the select symbol is reserved`
P.S. `async` is not needed here

Reply:
done